File: boj/b_2217.ts
function sol2217(n: number, arr: Array<number>) {
  arr.sort((a, b) => b - a);
  const result = [];
  for (let i = 0; i < n; i += 1) result.push(arr[i] * (i + 1));
  console.log(Math.max(...result));
}
sol2217(2, [10, 15]);

File: boj/b_2217.py
import sys
sys.stdin = open("input.txt", "r") # 제거
input = sys.stdin.readline

n = int(input())
arr = sorted([int(input()) for _ in range(n)], reverse=1)
result = []
for i in range(n):
    result.append(arr[i] * (i + 1))
print(max(result))